File: nathan-wilson/day-01/tip-calculator.js
function calculateTip(amount, rating) {
  let lowerCaseRating = rating.toLowerCase();

  switch (lowerCaseRating) {
    case 'terrible':
      return 0;
    case 'poor':
      return Math.ceil(amount * 0.05);
    case 'good':
      return Math.ceil(amount * 0.1);
    case 'great':
      return Math.ceil(amount * 0.15);
    case 'excellent':
      return Math.ceil(amount * 0.2);
    default:
      return 'Rating not recognised';
  }
}

// amount of check, rating which is a string that refers to tip percentage
// return the amount you tip
// 20 total amount, rating excellent which is 20% tip, return 4
// could return wrong value, could be case sensitive
// for each rating convert it to lowercase and then get whatever tip percentage it correlates to
// take the amount for each times 0.(whatever the tip amount is
// I can do this, its okay if it isnt the first try!

File: owen-coombs/day-27/Tip-Calculator.py
import math
def calculate_tip(amount, rating):
    tip = 0
    if rating.lower() == 'terrible':
        tip = 0
    elif rating.lower() == 'poor':
        tip = 0.05 * amount
    elif rating.lower() == 'good':
        tip = 0.10 * amount
    elif rating.lower() == 'great':
        tip = 0.15 * amount
    elif rating.lower() == 'excellent':
        tip = 0.20 * amount
    else:
        return 'Rating not recognised'
        
    
    return math.ceil(tip)

# use if elif 
# use .lower for case sensitive
